package.json

```diff
@@ -66,5 +66,10 @@
   },
   "engines": {
     "node": ">= 18 < 19"
+  },
```

Comment:
I would delete all mentions of 'enzyme' in the project and package.json. Then, try to delete your packages, install them and run tests.